src/eradiate/scenes/spectra/_solar_irradiance.py

```diff
@@ -80,7 +80,7 @@ class SolarIrradianceSpectrum(Spectrum):
         doc="Dataset identifier. Allowed values: see "
         ":attr:`solar irradiance dataset documentation <eradiate.data.solar_irradiance_spectra>`.",
         type="str",
-        default='``"thuiller_2003"``',
+        default="thuiller_2003",
```

Comment:
Let's make it clear that it's a string:
```suggestion
        default='"thuiller_2003"',
```